pandas/tests/io/test_compression.py

```diff
@@ -252,6 +252,24 @@ def test_gzip_compression_level(obj, method):
         compressed_size_fast = os.path.getsize(path)
         assert compressed_size_default < compressed_size_fast
 
+@pytest.mark.parametrize("method", ["to_pickle", "to_json", "to_csv"])
+def test_xz_compression_level_read(obj, method):
+    with tm.ensure_clean() as path:
+        getattr(obj, method)(path, compression="xz")
+        compressed_size_default = os.path.getsize(path)
+        getattr(obj, method)(path, compression={"method": "xz", "preset": 1})
+        compressed_size_fast = os.path.getsize(path)
+        assert compressed_size_default < compressed_size_fast
```

Comment:
Great that you found a way to test the forwarded arguments directly! In that case, you can probably remove the `test_xz_compression_invalid_args`